src/ansys/dpf/core/geometry_factory.py

```diff
@@ -0,0 +1,133 @@
+"""Geometry factory module containing functions to create the different geometries."""
+
+import numpy as np
+
+from ansys.dpf.core.geometry import Points, Line, Plane
+
+
+def create_points(coordinates, server=None):
+    """Construct points given its coordinates."""
+    return Points(coordinates, server)
+
+
+def create_line_from_points(points, num_points=100, server=None):
+    """Construct line from two DPF points."""
+    return Line(points, num_points, server)
+
+
+def create_line_from_vector(ini, end=None, num_points=100, server=None):
+    """Construct line from origin's coordinates and a vector direction."""
+    # Input check
+    if isinstance(ini[0], list):
+        if not len(ini) == 2:
+            raise ValueError("Exactly two points must be passed to define the vector")
+        if not len(ini[0]) == len(ini[1]) == 3:
+            raise ValueError(
+                "Each point must contain three coordinates 'x', 'y' and 'z'."
+            )
+        vect = ini
+    else:
+        if not len(ini) == 3:
+            raise ValueError(
+                "'ini' argument must be of length = 3 \
+            representing the 3D coordinates of the origin of the vector."
+            )
+        if not len(end) == 3:
+            raise ValueError(
+                "'end' argument must be of length = 3 \
+            representing the 3D coordinates of the end point of the vector."
+            )
+        vect = [ini, end]
+
+    return Line(vect, num_points, server)
+
+
+def create_plane_from_center_and_normal(center, normal, server=None):
+    return Plane(center, normal, server)
+
+
+def create_plane_from_points(points, server=None):
+    """Create plane from three points."""
+    # Input check
+    if isinstance(points, Points):
+        if not len(points) == 3:
```

Comment:
Makes me thing if we should have a method like ``point.is_in_plane(plane)`` or better ... ``point in plane``

Reply:
That could be useful, but I haven't seen where to use it yet.